pkgs/applications/office/beamerpresenter/default.nix

```diff
@@ -27,13 +28,13 @@
 
 stdenv.mkDerivation rec {
   pname = "beamerpresenter";
-  version = "0.2.3-1";
+  version = "0.2.4";
 
   src = fetchFromGitHub {
     owner = "stiglers-eponym";
     repo = "BeamerPresenter";
-    rev = "dd41a00b3c6c8b881fa62945165c965634df66f0";
-    sha256 = "11yj1zl8hdnqbynkbyzg8kwyx1jl8c87x8f8qyllpk0s6cg304d0";
+    rev = "v${version}";
+    sha256 = "1zypzmkiwc14b9224nr37y7wviasirmwhfmifb7q1b33877g41ji";
```

Comment:
```suggestion
    hash = "1zypzmkiwc14b9224nr37y7wviasirmwhfmifb7q1b33877g41ji";
```

Reply:
I'm not so familiar with Nix, but should it be
```
hash = "sha256-UQbyzkFjrIDPcrE6yGuOWsXNjz8jWyJEWiQwHmf91/8="
```
?
With your suggested changes I get this error:
error: hash '1zypzmkiwc14b9224nr37y7wviasirmwhfmifb7q1b33877g41ji' does not include a type, nor is the type otherwise known from context

Reply:
> I'm not so familiar with Nix, but should it be
> 
> ```
> hash = "sha256-UQbyzkFjrIDPcrE6yGuOWsXNjz8jWyJEWiQwHmf91/8="
> ```
> 
> ?

Yes exactly.